### PR TITLE
Fenix support

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/manifest_fenix.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  },
+
+  "icons": {
+    "48": "img/icon.svg",
+    "96": "img/icon.svg"
+  },
+
+  "incognito": "spanning",
+
+  "permissions": [
```
Comment (colintheshots) [inline src/manifest_fenix.json]: GeckoView will crash with the `captivePortal` or `identity` permission in the manifest.

File: src/manifest_fenix.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      "id": "secure-proxy@mozilla.com",
+      "strict_min_version": "68.0.1",
+      "update_url": "https://private-network.firefox.com/files/updates.json"
+    }
+  },
+
+  "homepage_url": "https://private-network.firefox.com",
+
+  "description": "__MSG_extensionDescription__",
+  "browser_action": {
```
Comment (colintheshots) [inline src/manifest_fenix.json]: I don't believe this browser_action can be included just yet with GeckoView. Support is coming for it though.

File: src/manifest_fenix.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
+  "manifest_version": 2,
```
Comment (colintheshots) [inline src/manifest_fenix.json]: Do we need to bump the manifest_version on changes?

Comment (bakulf) [reply src/manifest_fenix.json]: :)

File: src/background/mobileEvents.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import {Component} from "./component.js";
+import {constants} from "./constants.js";
+
+export class MobileEvents extends Component {
+  async init() {
+    if (!constants.isAndroid) {
```
Comment (pdehaan) [inline src/background/mobileEvents.js]: Will `constants.isAndroid` be defined yet, or do we need to explicitly call `await constants.init();` first?
Or does it only need to be initialized once, which is already done in src/background/main.js above?

File: src/manifest_fenix.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "page": "background/page.html"
+  },
+
+  "content_scripts": [
+    {
+      "matches": ["https://private-network.firefox.com/*"],
+      "js": ["content/fetch-fxa-flow-params.js"]
+    }
+  ]
+}
```
Comment (pdehaan) [inline src/manifest_fenix.json]: This is largely duplicated from the ./src/manifest.json, but I'm not sure of a better way of handling this, apart from regenerating the src/manifest_fenix.json file via some script which adds/removes the keys we don't want.

```diff
diff src/manifest.json src/manifest_fenix.json
18,23d17
<   "browser_action": {
<     "browser_style": true,
<     "default_icon": "img/badge_off.svg",
<     "default_title": "__MSG_extensionName__",
<     "default_popup": "popup/popup.html"
<   },
34,36c28,29
<       "browserSettings",
<       "captivePortal",
<       "identity",
---
>       "geckoViewAddons",
>       "nativeMessaging",
41d33
<       "telemetry",
```

**Pull request merged**